jupyterhub/tests/test_services_auth.py

```diff
@@ -217,7 +219,8 @@ def finish_thread():
         assert auth.login_url in r.headers['Location']
 
 
-def test_service_cookie_auth(app, mockservice_url):
+def test_hubauth_cookie(app, mockservice_url):
```

Comment:
I'm not sure it would help in this particular test case, but in cases where it would, parameterize is a good idea.